maps/map07.txt

```diff
@@ -0,0 +1,10 @@
+##########
+#o.     ##
+ ######  
+      ####
+#### ###..
+   # #. #
+##    #.##
+   #  #..#
+...#.....#
+##########
```

Comment:
Are you sure *this* map is useful? Maybe we shouldn't repeat this boring "tutorial" map again?